cmd/mircat/main.go

```diff
@@ -0,0 +1,94 @@
+package main
+
+import (
+	"github.com/AlecAivazis/survey/v2"
+	"github.com/pkg/errors"
+	"gopkg.in/alecthomas/kingpin.v2"
+	"os"
+)
+
+type arguments struct {
+	srcFile           *os.File
+	includedEvents    []string
+	includedIssEvents []string
+}
+
+var (
+	eventList = []string{
```

Comment:
This list could, maybe, be created dynamically by traversing the event log once and collecting the types of events that occur. Like this, this list would not have to be adapted each time new events are added.